src/test_l1_dsVideoDevice.c

```diff
@@ -890,20 +932,28 @@ void test_l1_dsVideoDevice_positive_dsGetVideoCodecInfo(void)
     DS_ASSERT_AUTO_TERM_NUMERICAL((handle >= 0), true);
 
     // Step 03: Get video codec information with dsVIDEO_CODEC_MPEGHPART2
+    #ifdef ENABLE_ENHANCED_ERROR_CODE
```

Comment:
write else case and check for dsERR_OPERATION_NOT_SUPPORTED
as we mentation in the excl (dsGetVideoCodecInfo_L1_positive)

Reply:
done